packages/feature-detector/src/has-edge-signature.ts

```diff
@@ -0,0 +1,24 @@
+import { Project } from 'ts-morph'
+import { getDefaultExport } from './utils/exports'
+import { getReturnType } from './utils/functions'
+import { buildProject } from './utils/project'
+import { extractFromPromise, isSubClassOf } from './utils/types'
+
+/**
+ * Returns true if the default export of the provided file is a function returning a web Response object.
```

Comment:
I like this description as resume of the package behavior 👌

Reply:
Thanks, but unfortunately, this package will do more: it'll also publish another function to list globals and 3rd party module used (see [linear](https://linear.app/vercel/issue/EC-544/build-a-minimal-ts-morph-feature-detector))